services/app-api/forms/naaar.json

```diff
@@ -83,12 +83,64 @@
               },
               {
                 "id": "reportingScenario",
-                "type": "text",
-                "validation": "textOptional",
+                "type": "radio",
+                "validation": "radio",
                 "props": {
-                  "label": "I.A.5 Reporting scenario (TODO)",
-                  "hint": "TODO: Convert to radio",
-                  "disabled": true
+                  "label": "1.A.5 Reporting scenario",
+                  "hint": "<p>Enter the scenario under which the state is submitting this form to CMS. Under 42 C.F.R. § 438.207(c) - (d), the state must submit an assurance of compliance after reviewing documentation submitted by a plan under the following three scenarios:</p><ul><li>Scenario 1: At the time the plan enters into a contract with the state;</li><li>Scenario 2: On an annual basis;</li><li>Scenario 3: Any time there has been a significant change (as defined by the state) in the plan's operations that would affect its adequacy of capacity and services, including (1) changes in the plan's services, benefits, geographic service area, composition of or payments to its provider network, or (2) enrollment of a new population in the plan.</li></ul><br><p>States should complete one (1) form with information for applicable managed care plans and programs. For example, if the state submits this form under scenario 1 above, the state should submit this form only for the managed care plan (and the applicable managed care program) that entered into a new contract with the state. The state should not report on any other plans or programs under this scenario. As another example, if the state submits this form under scenario 2, the state should submit this form for all managed care plans and managed care programs.</p>",
```

Comment:
Could toss a `<br>` in `scenarios:</p><br><ul><li>Scenario 1` to give more space for readability.

Reply:
I have some suggestions for cleaning up this hint text a bit, but because all of this content is WIP, I don't think it should block the PR.